primo/data_parser/default_data.py

```diff
@@ -0,0 +1,260 @@
+#################################################################################
+# PRIMO - The P&A Project Optimizer was produced under the Methane Emissions
+# Reduction Program (MERP) and National Energy Technology Laboratory's (NETL)
+# National Emissions Reduction Initiative (NEMRI).
+#
+# NOTICE. This Software was developed under funding from the U.S. Government
+# and the U.S. Government consequently retains certain rights. As such, the
+# U.S. Government has been granted for itself and others acting on its behalf
+# a paid-up, nonexclusive, irrevocable, worldwide license in the Software to
+# reproduce, distribute copies to the public, prepare derivative works, and
+# perform publicly and display publicly, and to permit others to do so.
+#################################################################################
+
+# Standard libs
+from dataclasses import dataclass
+from typing import Union
+
+# This file should contain all the constants
+FEASIBILITY_TOLERANCE = 1e-6  # Optimization Feasibility tolerance
+EARTH_RADIUS = 3959.0  # Earth's radius in Miles
+CENSUS_YEAR = 2020
+CONVERSION_FACTOR = 5.614583  # convert bbl of oil to mcf of gas
+BING_MAPS_BASE_URL = "http://dev.virtualearth.net/REST/V1/Routes/Driving"
+# Fixed start coordinates in elevation util to get nearest road point
+START_COORDINATES = (40.44, -79.94)
+
+
+# Set of supported impact metrics along with
+# the required data for the analysis.
+@dataclass()
+class _SupportedContent:
+    name: str
+    full_name: str
+    has_submetrics: bool = False
+    is_submetric: bool = False
+    parent_metric: Union[str, None] = None
```

Comment:
Do we need a post_init method, specifically for validating that a parent_metric specified exists? Similarly if has_submetrics is specified as True, do we need to check that they exist?